utils/edit.cc

```diff
@@ -68,6 +70,10 @@ static int parseCommandLine(app_options& opts, int argc, char const* const* argv
             opts.replace[1] = optarg;
             break;
 
+        case 's':
```

Comment:
This should be listed in `utils/transmission-edit.1` as well

Reply:
Ah I missed that too. Done.